### PR TITLE
Feature: string values are now html un-escaped

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

File: scripts/tests/test_ejpcsv2json.py [hunk 1]
```diff
@@ -34,3 +34,16 @@ def test_standard_parse_with_filename():
         {'poa_m_ms_id': '38546', 'poa_m_ms_no': '27706', 'poa_r_received_dt': '2017-04-11 15:38:47.637', 'poa_r_receipt_dt2': '2017-06-01 03:31:21.817', 'date_generated': '2018-11-18', 'imported_timestamp': '2018-11-01T23:59:59Z', 'provenance': {'source_filename': filename}},
         {'poa_m_ms_id': '42025', 'poa_m_ms_no': '30325', 'poa_r_received_dt': '2017-07-11 08:08:57.670', 'poa_r_receipt_dt2': '2017-07-26 05:14:54.213', 'date_generated': '2018-11-18', 'imported_timestamp': '2018-11-01T23:59:59Z', 'provenance': {'source_filename': filename}}]
     assert expected == actual
+
+def test_unescape_html_escaped_values():
+    row = {
+        'foo': "L&#x00E1;szl&#x00F3; Csan&#x00E1;dy",
```
Comment (de-code) [inline scripts/tests/test_ejpcsv2json.py:57]: Might be best to redact the name. e.g. just include the special characters.

File: scripts/ejpcsv2json.py [hunk 2]
```diff
@@ -30,6 +30,13 @@ def first(lst):
     except IndexError:
         return None
 
+def unescape_html_escaped_values(row):
+    "mutator"
+    for key, val in row.items():
+        if isinstance(val, str):
+            row[key] = html.unescape(val)
```
Comment (de-code) [inline scripts/ejpcsv2json.py:44]: Personally prefer functional over mutator function. Might be okay in a small script like this one.

@giorgiosironi @GiancarloFusiello ?

**Review by de-code — APPROVED** [review]

File: scripts/ejpcsv2json.py (hunk 2 above)
Comment (giorgiosironi) [reply scripts/ejpcsv2json.py:44]: could be copied over to a new dictionary, as a mutator returning the result is misleading (if it's mutating it why returning it at all?)

Comment (lsh-0) [reply scripts/ejpcsv2json.py:44]: > Personally prefer functional over mutator

opposite of mutable would immutable, not functional.

**Pull request merged**